site/content/integrate/apps/quickstart/quick-start-serverless/_index.md

```diff
@@ -1,436 +1,93 @@
-	"requested_permissions": [
-		"act_as_bot"
-	],
-	"requested_locations": [
-		"/command"
-	],
-	"http": {
-		"root_url": "http://localhost:8080"
-	},
-	"aws_lambda": {
-		"functions": [
-			{
-				"path": "/",
-				"name": "hello-serverless",
-				"handler": "hello-serverless",
-				"runtime": "go1.x"
-			}
-		]
-	},
-	"open_faas": {
-		"functions": [
-			{
-				"path": "/",
-				"name": "hello"
-			}
-		]
-	}
-}
+Install via /apps install http http://mattermost-apps-golang-serverless:8080/manifest.json 
 ```
 
-### 2. Static (`./static`)
-
-The example app will use an icon, you can download an example icon using:
-
-```bash
-mkdir ./static
-curl https://github.com/mattermost/mattermost-plugin-apps/raw/master/examples/go/hello-serverless/static/icon.png -o static/icon.png
-```
-
-### 3. `./function` package
-
-#### `bindings.json`
-```json
-{
-	"type": "ok",
-	"data": [
-		{
-			"location": "/command",
-			"bindings": [
-				{
-					"icon": "icon.png",
-					"label": "hello-serverless",
-					"description": "Hello Serverless app",
-					"hint": "[send]",
-					"bindings": [
-						{
-							"location": "send",
-							"label": "send",
-							"call": {
-								"path": "/send"
-							}
-						}
-					]
-				}
-			]
-		}
-	]
-}
-```
-
-#### `send_form.json`
-
-```json
-{
-	"type": "form",
-	"form": {
-		"title": "Hello, serverless!",
-		"icon": "/static/icon.png",
-		"fields": [
-			{
-				"type": "text",
-				"name": "message",
-				"label": "message"
-			}
-		],
-		"call": {
-			"path": "/send"
-		}
-	}
-}
-```
-
-### `function.go`
-
-```go
-package function
-
-import (
-	_ "embed"
-	"encoding/json"
-	"fmt"
-	"net/http"
-
-	"github.com/mattermost/mattermost-plugin-apps/apps"
-	"github.com/mattermost/mattermost-plugin-apps/apps/appclient"
-	"github.com/mattermost/mattermost-plugin-apps/utils/httputils"
-)
-
-//go:embed bindings.json
-var bindingsData []byte
-
-//go:embed send_form.json
-var formData []byte
-
-// Handler for OpenFaaS and faasd.
-func Handle(w http.ResponseWriter, r *http.Request) {
-	InitApp(apps.DeployOpenFAAS)
-	http.DefaultServeMux.ServeHTTP(w, r)
-}
-```
+## Install the app on Mattermost
 
-Note: names `package function` and `func Handle` appear to be required
-(hardcoded) for OpenFaaS.
+Next, access your development Mattermost server at [http://localhost:8065](http://localhost:8065) and use the `/apps install http http://mattermost-apps-golang-serverless:8080/manifest.json ` slash command to install the webhooks app. Select `Agree to grant the app access to APIs and Locations` and click `Submit` to finish the installation.
 
-`golang-middleware` template makes use of adding boilerplate handlers to
-`http.DefaultServeMux`, so we just need to add our handlers and serve.
+## Use the app
 
-```go
-var deployType apps.DeployType
+You can now use the `/hello-serverless send` command with an optional `message` argument. This will cause the app's bot user to DM you a response:
```

Comment:
```suggestion
You can now use the `/hello-serverless send` command with an optional `message` argument. This will cause the App's bot user to DM you a response:
```